synapse/tests/test_lib_ingest.py

```diff
@@ -429,6 +429,59 @@ def test_ingest_cast(self):
 
             self.nn(core.getTufoByProp('strform', 'lulz'))
 
+    def test_ingest_jsoncast(self):
+
+        # similar to csv data...
+        data = ['vertex.link/pennywise', '2017/10/10 01:02:03', 'served:balloon', '1.2.3.4', 'we all float down here']
```

Comment:
Can you add another item here that is a dictionary w/ the json cast?

Reply:
done